lms/views/exceptions.py

```diff
@@ -60,6 +60,20 @@ def reused_consumer_key(self):
 
         return {}
 
+    @exception_view_config(
+        context=SerializableError, renderer="lms:templates/error_dialog.html.jinja2"
+    )
+    def serializable_error(self):
```

Comment:
It kind of doesn't, as that one catches things in the API space specifically, and this one catches all other view exceptions. 

I suppose I could bump the `SerializableError` down into `views.exceptions` instead of `views.api.exceptions`?